README.md

```diff
@@ -2,25 +2,284 @@
 
 Specular is a library for building Web-based UIs in PureScript, based on
 Functional Reactive Programming (FRP). It is heavily inspired by
-[Reflex][reflex] and [Reflex-DOM][reflex-dom] and supports a similar programming
-model.
+[Reflex][reflex] and [Reflex-DOM][reflex-dom].
 
-## Getting started
+## API
 
-There's a tutorial currently in the making: [Tutorial (incomplete)](./doc/Tutorial.md)
+### FRP types
 
-Currently there's no tutorial or introductory documentation. However, there's
-plenty of material about Reflex-Dom. See
-[A Beginner-friendly Step by Step Tutorial for Reflex-Dom](https://github.com/hansroland/reflex-dom-inbits/blob/master/tutorial.md)
-for a good introduction to the programming model. It should be adapted to
-PureScript and Specular in the near future.
+To use Specular effectively, you need to be familliar with some basic types.
 
-There are some [examples](test/browser/examples) that are also integration
-tests. They can be viewed in a Web browser:
+#### [Dynamic](https://pursuit.purescript.org/packages/purescript-specular/docs/Specular.FRP.Base#t:Dynamic)
 
+`Dynamic a` represents a read-only reference to a changing value of type `a`.
+
+```purescript
+-- | Read the current value of a `Dynamic`.
+readDynamic :: forall m a. MonadEffect m => Dynamic a -> m a
+
+-- | Execute the given action for the current value, and each new value when it changes.
+subscribeDyn_ :: forall m a. MonadEffect m => MonadCleanup m => (a -> Effect Unit) -> Dynamic a -> m a
```

Comment:
Bike shedding: inconsistency in naming functions that end with `dynamic` ?  We have:  `Dyn`, `Dynamic`, `D`